packages/slate/src/changes/by-key.js

```diff
@@ -12,6 +12,26 @@ import Node from '../models/node'
 
 const Changes = {}
 
+/**
+ * Obtains the normalization setting to use for the change based on the
+ * change object flags and the change operation options.
+ *
+ * @param {Object} flags - flags object for the current change object
+ * @param {Object} options - options for the change operation
+ * @return {bool} - the normalization configuration to use. Defauls to true
+ *   if flags or options does not have a normalization configuration
+ */
+
```

Comment:
order is important here because we do not want the user to override the normalize flag with { normalize: false }. This may be a debatable decision, but this achieves the desirable effect of withMutations having full suppression of normalization within its scope. Allowing options.normalize to override flags would allow the user to enable normalization early within withMutations and break the contract of that function/create defects